base/applications/mspaint/mouse.cpp

```diff
@@ -205,20 +205,13 @@ struct RectSelTool : ToolBase
     }
 };
 
-// TOOL_RUBBER
-struct RubberTool : ToolBase
+struct GenericDrawTool : ToolBase
```

Comment:
Yes that's nice, I wanted to suggest that next.

Reply:
I think a similar idea might be done for selection-like tools (see whether this could be useful or not)

Reply:
Done.